addons/tmf_loadouts/loadouts/UK_Army_2005.hpp

```diff
@@ -218,7 +218,7 @@ class dm : r {
 	displayName = "Designated Marksman";
 	primaryWeapon[] = {"CUP_arifle_L86A2"};
 	scope[] = {"cup_optic_susat"};
-	secondaryWeapon[] = {"hlc_pistol_P226US"};
+	secondaryWeapon[] = {"hlc_pistol_P226US"};
```

Comment:
sidearmWeapon

Reply:
Uhhh, fix'd I think? No clue how that happened.